src/css/base.css

```diff
@@ -62,7 +62,7 @@
   */
   --text-light: #ffffff;
   --text-dark: #403F3F;
-  --text-accent:#B6D880;
+  --text-accent:#004cffa2;
```

Comment:
I like the green we have on hover, e.g. with our names on Cohort X. Maybe we could make the links the blue that's better for color contrast but keep the green when you hover.